src/app/view/components/main-legacy/main-legacy.component.ts

```diff
@@ -11,26 +9,13 @@ import { DotCustomEventHandlerService } from '@services/dot-custom-event-handler
     templateUrl: './main-legacy.component.html'
 })
 export class MainComponentLegacyComponent implements OnInit {
-    constructor(
-        private dotRouterService: DotRouterService,
-        private dotIframeService: DotIframeService,
-        private dotCustomEventHandlerService: DotCustomEventHandlerService
-    ) {}
+    constructor(private dotCustomEventHandlerService: DotCustomEventHandlerService) {}
 
     ngOnInit(): void {
         document.body.style.backgroundColor = '';
         document.body.style.backgroundImage = '';
     }
 
-    /**
-     * Reload content search iframe when contentlet editor close
-     *
-     * @memberof MainComponentLegacyComponent
-     */
-    onCloseContentletEditor(): void {
-        this.dotIframeService.reloadData(this.dotRouterService.currentPortlet.id);
-    }
-
```

Comment:
Why?

Reply:
@fmontes 
Currently when you add a new content in Content --> Search, and when you close the dialog, the code that handles the **Reload of the data** is only present on the `<dot-main-component> (main-legacy.component.ts)` which is the parent component of `<dot-create-contentlet>`

when you put the new Url to create content in the browser, the component that gets loaded is <dot-create-contentlet> so it does not have any parent component, and when the dialog gets closed no data is being reloaded

so I moved the logic that reloads the data from the parent `<dot-main-component> (main-legacy.component.ts)` to the dialog `<dot-create-contentlet>` in that way the data gets reloaded on both scenarios

Reply:
Gracias!